portal/templates/utilisation-report-service/record-correction/provide-utilisation-report-correction.njk

```diff
@@ -0,0 +1,44 @@
+{% extends "index.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
+{% import "./_macros/correction-request-details-table.njk" as correctionRequestDetailsTable %}
+
+{% block pageTitle %}Record correction{% endblock %}
+
+{% block content %}
+
+    <h1 class="govuk-heading-xl">
+      Report GEF utilisation and fees
+    </h1>
+
+    <h2 class="govuk-heading-l">
+      Record correction
+    </h2>
+
+    {{ correctionRequestDetailsTable.render({ details: correctionRequestDetails }) }}
+
+    <div class="govuk-button-group">
```

Comment:
No behaviour added to these buttons, just added the placeholders here - behaviour covered by a future ticket